app/models/flood_risk_engine/enrollment_exemption.rb

```diff
@@ -20,24 +20,28 @@ def self.include_long_dredging?
       includes(:exemption).where(flood_risk_engine_exemptions: { code: Exemption::LONG_DREDGING_CODES }).any?
     end
 
-    def latest_decision
+    def accept_reject_decision_user
+      User.find_by_id(accept_reject_decision_user_id).try(:email)
```

Comment:
Yes, I would have put it in a decorator in BO but couldn't be bothered dealing with the ensuing comments.  I guess that engine is never going be used for another app, so from pragmatic perspective coupling not really an issue